### PR TITLE
fix: openapi-validation test for quorum plugin

### PR DESCRIPTION
**Pull request by elenaizaguirre**
Includes tests for endpoints deployContractSolBytecodeV1,
invokeContractV1 and runTransactionV1, each of them
with test cases:
  - Right request
  - Request including an invalid parameter
  - Request without a required parameter

Closes: #1286

Relationed with #847 

Signed-off-by: Elena Izaguirre <e.izaguirre.equiza@accenture.com>

### CODE REVIEW

File: packages/cactus-plugin-ledger-connector-quorum/src/main/json/openapi.json [hunk 1]
```diff
@@ -710,6 +693,75 @@
                     }
                 }
             }
+        },
+        "/api/v1/plugins/@hyperledger/cactus-plugin-consortium-manual/consortium/jws": {
```
Comment (elenaizaguirre) [inline packages/cactus-plugin-ledger-connector-quorum/src/main/json/openapi.json:697]: @petermetz I had to add these endpoints to Quorum openapi document because test api-client-routing-node-to-node.test.ts from cactus-test-api-client didn't working. I think it was trying to call consortium-manual endpoints through Quorum API client and validation was failing.
Is there any way to do it better?

**Review by petermetz — CHANGES_REQUESTED** [review]
@elenaizaguirre Same requests/questions here as in the other OpenAPI validation PR

File: packages/cactus-plugin-ledger-connector-quorum/src/main/json/openapi.json (hunk 1 above)
Comment (petermetz) [reply packages/cactus-plugin-ledger-connector-quorum/src/main/json/openapi.json:697]: @elenaizaguirre It's most likely being caused by the open API validator trying to look up the validation metadata of the consortium manual plugin and not finding it, showing screenshots below with how I debugged into it.
One more tip: If you want to see what request handlers are registered in an express app you can run `app._router.stack` on the debug console to print it like I did (see other screenshot)

TLDR solution: you might need to tie back together the consortium manual plugin PR and the quorum connector PR because of this dependency.

Screenshot 2021-09-01 141141
Screenshot 2021-09-01 140852
Screenshot 2021-09-01 140009

**Review by petermetz — APPROVED** [review]
LGTM

**Review by takeutak — APPROVED** [review]
